spec/features/add_unassigned_nodes_feature_spec.rb

```diff
@@ -47,6 +47,9 @@
 
   scenario "An user check all nodes at once to be added", js: true do
     using_wait_time 5 do
+      # wait for all minions to be there
```

Comment:
I don't want to conflict with your fixing flaky tests branch so I told Vítor I was going to fix some stuff trying to change the minimum from other places so we don't conflict a lot. It's redundant, and `using_wait_time` should be removed almost everywhere :)